fix: issue 368- consistent namespace scope resolution

PaurushGarg opened the pull request:
Fixes Issue (368)
In .cc files across whole project, opentelemetry::* is simplified to * , to follow the namespace guidelines. To do this, following
namespace aliases were used, to establish convention:
- namespace common = opentelemetry::common;
- namespace nostd = opentelemetry::nostd;
- namespace trace = opentelemetry::trace;
- namespace trace_api = opentelemetry::trace;
- namespace resource = opentelemetry::sdk::resource;
- namespace exporter_trace = opentelemetry::exporter::trace;
- namespace context = opentelemetry::context;
- namespace metrics_api = opentelemetry::metrics;
- namespace sdkmetrics  = opentelemetry::sdk::metrics;
- namespace exportermetrics = opentelemetry::exporter::metrics;
- namespace trace_sdk = opentelemetry::sdk::trace;
- namespace sdktrace = opentelemetry::sdk::trace;
- namespace exporterlogs  = opentelemetry::exporter::logs;
- namespace exportertrace = opentelemetry::exporter::trace;
- namespace proto = opentelemetry::proto;
- namespace context     = opentelemetry::context;
- namespace otlp     = opentelemetry::exporter::otlp;
- namespace plugin    = opentelemetry::plugin;
- namespace curl = opentelemetry::ext::http::client::curl;
- namespace http_client = opentelemetry::ext::http::client;
- namespace logs_api    = opentelemetry::logs;
- namespace sdklogs  = opentelemetry::sdk::logs;

**Note:** _trace and trace_api both are used for opentelemetry::trace;
 trace_sdk and sdktrace both are used for opentelemetry::sdk::trace; These were already existing in the code._


Please provide a brief description of the changes here.
Usage of scope resolution was made consistent across all files using namespace convention and guidelines.

ThomsonTan commented:
@PaurushGarg let us know if you will get a chance to review all the comments and resolve the conflict, or need our help on it.

PaurushGarg commented:
> @PaurushGarg let us know if you will get a chance to review all the comments and resolve the conflict, or need our help on it.

@ThomsonTan Sorry it took some time. I have updated both the comments regarding trace_sdk and metric_sdk, and also updated the header files to removes the namespace aliases.

ThomsonTan commented:
@lalitb do you have any other comments?

lalitb commented:
> @lalitb do you have any other comments?

No, it looks good now. Thanks, @PaurushGarg for your efforts.
